### PR TITLE
Dräger add

### PR DESCRIPTION
**Pull request by barde**
Lokal erfolgreich getestet.

Die gitignore habe ich ergänzt, da vermutlich noch mehr Lizenzbedingungen kommen und ich nicht mit dem Notebook meiner IT-Kollegen arbeiten kann. 🤯

### CODE REVIEW

File: products.json [hunk 1]
```diff
@@ -322,6 +322,44 @@
...
+                },
+                "children": [
+                    {
+                        "text": "Software",
+                        "state": {
+                            "opened": false
+                        },
+                        "children": [
+                            {
+                                "text": "Dräger add",
```
Comment (nathanhi) [inline products.json:339]: Isn't the product called "Dräger DrugCheck"?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (nathanhi) [inline .gitignore:1]: Maybe generate a more complete .gitignore using gitignore.io?

File: products.json (hunk 1 above)
Comment (barde) [reply products.json:339]: The product will be called "add" while DrugCheck will be the first components in the system. Right now, there is another part in development and one other in planning.

File: .gitignore (hunk 2 above)
Comment (barde) [reply .gitignore]: Done!

**Comment by nathanhi** [comment]
Feel free to merge. You'll need to sign your commits though. :)

**Comment by barde** [comment]
Had to rebase and force push as the signing key was from another key.

**Comment by nathanhi** [comment]
> Had to rebase and force push as the signing key was from another key.

Only one of two commits is GPG signed though!

**Comment by barde** [comment]
Finally all signed! Thanks for the help, Nathan!

**Pull request merged**